core/http/src/known_media_types.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! known_media_types {
         MOV (is_mov): "quicktime video", "video", "quicktime",
         MP4 (is_mp4): "MPEG4 Video", "video", "mp4",
         ZIP (is_zip): "ZIP archive", "application", "zip",
+        RON (is_ron): "Rusty Object Notation", "text", "plain",
```

Comment:
This isn't the right way to go about this; nothing should change in `known_media_types.rs` at all since there's no media type for RON. Checks against RON should probably either require/expect `text/plain` or preferably not check the MIME type at all, and code that outputs RON should either say it is `text/plain` or not make any claim.